src/EfficientDynamoDb/Operations/ExecuteStatement/ExecuteStatementResponse.cs

```diff
@@ -0,0 +1,48 @@
+﻿using EfficientDynamoDb.DocumentModel;
+using EfficientDynamoDb.Operations.Shared.Capacity;
+using System.Collections.Generic;
+
+namespace EfficientDynamoDb.Operations.ExecuteStatement
+{
+    public class ExecuteStatementResponse
+    {
+        /// <summary>
+        /// Gets and sets the property Items. 
+        /// <para>
+        /// If a read operation was used, this property will contain the result of the read operation;
+        /// a map of attribute names and their values. For the write operations this value will
+        /// be empty.
+        /// </para>
+        /// </summary>
+        public IReadOnlyList<Document>? Items { get; set; }
```

Comment:
Similarly to `QueryResponse` and `ScanResponse`, the `Items` property will never be null. When there are no items in the response it will be set to empty array. So the property shouldn't be nullable. To suppress the nullability error, you can use `= null!;` default value.

The same applies to transact and batch operations.